openedx_external_enrollments/tasks.py

```diff
@@ -20,3 +26,16 @@ def generate_salesforce_enrollment(data, *args, **kwargs):  # pylint: disable=un
     else:
         # Calling the controller enrollment method
         enrollment_controller._post_enrollment(data)  # pylint: disable=protected-access
+
+
+@periodic_task(run_every=timedelta(days=1))
+def pathstream_periodic_task():
```

Comment:
@Jacatove , doesn't this task need to have retry feature for failed executions?

Reply:
@ivanvgh thanks for the suggestion, I will add it.